src/renderer/api-decorator.js

```diff
@@ -444,8 +453,14 @@ limitations under the License.
             }, 1);
         });
 
-        var convertedOpts = convertOptionsToElectronSync(options);
-        fin.__internal_.downloadPreloadScripts(convertedOpts.preload, () => {
+        const convertedOpts = convertOptionsToElectronSync(options);
+        const preloadScriptsPayload = {
+            uuid: options.uuid,
+            name: options.name,
+            scripts: convertedOpts.preload
```

Comment:
**For future refactor (because we need to merge this asap):** Why start using `scripts` here when we are using `preload` most everywhere else?